core/server/config/overrides.json

```diff
@@ -64,5 +64,18 @@
     },
     "maintenance": {
         "enabled": false
+    },
+    "api": {
+        "versions": {
+            "active": {
+                "admin": "v2/admin",
+                "client": "v2/client"
+            },
+            "stable": {
+                "admin": "v0.1",
+                "client": "v0.1"
```

Comment:
@kirrg001 Should we add another option for each API version as `"default"` other than `"admin"` and `"client"` for future-proofing? Currently either one needs to pass explicit `admin: true` or it defaults to client. Can be also something which we can add later if needed though, wdyt?

Reply:
Hmm i think we can add this if we really need it?

Reply:
Yeah was thinking of same, no need for premature optimization as for near future we only have `content` and `admin`!